src/validation.h

```diff
@@ -122,6 +122,12 @@ static const unsigned int DEFAULT_BANSCORE_THRESHOLD = 100;
 static const bool DEFAULT_PERSIST_MEMPOOL = true;
 /** Default for -mempoolreplacement */
 static const bool DEFAULT_ENABLE_REPLACEMENT = true;
+/** Default for -mempoolreplacementtimeout in seconds after which transactions in mempool are replaceable (i.e. 6 hours) */
+static const int64_t DEFAULT_REPLACEMENT_TIMEOUT = 6 * 60 * 60;
+/** Default for buffer in seconds on top of the timeout after which transactions in mempool are replaceable (i.e. 6 hours) in the GUI */
+static const int64_t REPLACEMENT_TIMEOUT_BUFFER = 20 * 60;
```

Comment:
20 minutes seems a bit long to me.

How long do transactions take to propagate to miners these days? Surely more like a minute or two? In the replacement case it's ok if not every last miner actually gets the replacement; having the occasional one missing isn't a big deal.

How about five minutes?

Reply:
I think 5 minutes would be fine. @TheBlueMatt ?

Reply:
Could also use a code comment to explain why this additional timeout is needed.

Reply:
@Sjors added a comment